yarn.lock

```diff
@@ -4748,11 +4748,6 @@
   resolved "https://registry.yarnpkg.com/@types/deep-assign/-/deep-assign-0.1.1.tgz#c10aa2d6202774015f5820ed303b82ab3b0bde9b"
   integrity sha512-88jktQ7RJ96bJ4S8I2oeRylMXGeaZt/QBG0k8PiQ5UXCIliUQm0Pzlv3zcN1ZvEIuaLBgMkP0ynr8fAkwChgOQ==
 
-"@types/doctrine@0.0.3":
```

Comment:
We still use `doctrine` in `scripts/gulp/plugins/util/parseDocblock.ts` as I see. Do we really want to remove types?

Reply:
Good catch, I'll hoist the types to the root instead